src/Mod/TechDraw/Gui/TaskSelectLineAttributes.ui

```diff
@@ -120,7 +120,7 @@
      <item row="2" column="0">
       <widget class="QRadioButton" name="rbMiddle">
        <property name="text">
-        <string>middle 0,35</string>
+        <string>Middle 0,35</string>
```

Comment:
Even though I prefer European commas, then is it meant to be so?

Reply:
This needs to be fixed by using the line groups we have since a while in TD. @WandererFan , could you please have a look when you find time?

However, this PR is just to fix a regression -> merging

Reply:
What am I looking for?   Using the weights from the LineGroup file instead of hard coded?

Reply:
Yes, this would be my proposal. So instead of the radio buttons, a combobox to select the line group like we have in the TD preferences.

By the way, what menu do I have to use to see the TaskSelectLineAttributes dialog?